src/systems/generic/implementations/vanilla/generic-component-media-container/generic-component-media-container.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Hero Banner component
+ * @define hero-banner
+ */
+
+// Import base
+@import '@ecl/generic-base/generic-base';
+
+@mixin ecl-media-container() {
+  .ecl-media-container {
+    background-color: $ecl-color-grey-10;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+  }
+
+  .ecl-media-container__media {
+    display: block;
+    height: auto;
+    object-fit: cover;
+    width: 100%;
+  }
+
+  .ecl-media-container__caption {
+    font: $ecl-font-prolonged-m;
```

Comment:
I guess it is ok like this, but the specs makes no sense (they put prolonged in font-weight, and kept UI line height...)